cpp/src/plasma/client.cc

```diff
@@ -932,7 +932,13 @@ Status PlasmaClient::List(ObjectTable* objects) { return impl_->List(objects); }
 
 Status PlasmaClient::Abort(const ObjectID& object_id) { return impl_->Abort(object_id); }
 
-Status PlasmaClient::Seal(const ObjectID& object_id) { return impl_->Seal(object_id); }
+Status PlasmaClient::Seal(const ObjectID& object_id) {
+  return impl_->Seal(object_id, true);
+}
+
+Status PlasmaClient::SealWithoutNotification(const ObjectID& object_id) {
```

Comment:
Lets do `PlasmaClient::Seal(const ObjectID& object_id, bool notify = true)` (this is a question of taste, generally I prefer to keep the API simple and there not to be too many methods around so first time users don't need to think too hard about what to call; most users will use the default here I think).